content/posts/2023/reana-0.9.2.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "REANA 0.9.2 is released"
+date: 2023-12-18T09:00:00+01:00
+---
+
+REANA 0.9.2 has just been released! This is a minor update that removes the limit on the number of restarts of a workflow, brings some more performance improvements, provides initial support for ARM platforms, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### Increasing workflow restart limit
+
+REANA already offers the possibility to restart a workflow, that is to execute a new workflow whose workspace is shared with the previous one. This comes in handy when developing a new analysis, for example to test small changes to the code or to fix some issues.
+
+Up until now, a workflow could be restarted only up to nine times. This limitation has been removed with REANA 0.9.2 and it is now possible to restart workflows as many times as needed.
+
+```console
+$ reana-client restart -w reana-demo-root6-roofit.1.9
+==> SUCCESS: reana-demo-root6-roofit.1.10 is pending
+```
+
+### Performance improvements
+
+Similarly to the previous release, REANA 0.9.2 brings some more performance improvements.
+In particular, the usage of the database has been optimised in order to reduce the time needed to perform some of the most common operations, including listing and filtering workflows and jobs.
+
+These improvements can be quite noticeable, even more so when users have executed tens of thousands of workflows and jobs.
+For example, we have measured a performance improvement of around 18x (18 times, not percent!) when listing all the details of a thousand workflows with `reana-client-go`.
+
+### Launch-on-REANA badge generator
+
+REANA 0.9.0 has introduced the possibility to construct custom URLs which can be used to easily launch on REANA analyses that are hosted on remote publicly-accessible sources such as GitHub, GitLab or Zenodo.
```

Comment:
REANA 0.9.0 has introduced a possibility to launch workflows from remote publicly-accessible sources such as GitHub, GitLab, or Zenodo.  This is being done by constructing special launcher URL that can be then shared with other users and used to create clickable badges that can be embedded in the README files in your remote repositories.